var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -226,8 +226,14 @@ def common_config_args(self):
 
     @when('@3')
     def configure(self):
-        # A user reported that this was necessary to link Qt3 on ubuntu
-        os.environ['LD_LIBRARY_PATH'] += os.pathsep + os.getcwd() + '/lib'
+        # A user reported that this was necessary to link Qt3 on ubuntu.
+	# However, if LD_LIBRARY_PATH is not set the qt build fails, check
+	# and set LD_LIBRARY_PATH if not set, update if it is set.
+	if not os.environ.get('LD_LIBRARY_PATH'):
```

Comment:
The flake8 tests are crashing because you mixed tabs and spaces. Please only use spaces.